### ISSUE TITLE
Missing Movies in Library

### ISSUE THREAD

**Issue by SplitEnz** [ErsatzTV/ErsatzTV#697]
I have an issue with some movie titles not appearing on the Ersatz server. I had suspected some stuff was missing when I first installed the server a couple of weeks back, but couldn't track anything specific down at the time.
Today, I did a search for 'Pirates' as I knew I had all five of the 'Pirates of the Caribbean' movies in my local collection, but only 2 titles are showing on the server. The NFO files for all 5 movies look good as far as I can see.
Also, when I click on the 'Movies' heading of the media section, the server is stating I have 4964 movies. My movie library isn't anywhere near that figure, so something seems to have gone wrong at the scraping stage.
What would be the ramifications if I was to delete the current local movie path, then re-add the same path and scape again?
Would I lose all the other settings created, ie channels, schedules, collections, etc, etc once the current path is deleted?

**Comment by jasongdove** [comment]
Sorry for the delay on this - is this still an issue? If the count is incorrect, I would stop ErsatzTV, delete the `search-index` config subfolder and then restart ErsatzTV. It will then rebuild the search index (which is used for the media counts, among other things).

**Comment by SplitEnz** [comment]
Thanks for the hint. I did as you suggested, which did indeed reduce the total library count quite dramatically. However, when I did the same search for my 'Pirates of the Caribbean' movies, only 2 of the 5 were still showing. I then checked the total library count in Kodi and that seems to suggest that I'm still about 200 adrift. I had another quick look at the NFO files today for the above mentioned movies, but again everything looks good as far as I can see.
Is there anything more I can do to completely refresh the movie path to try and pick up these missing entries?
If not, It's not a huge deal, particularly as everything has been working wonderfully on the channels I do have created.
